README.md

```diff
@@ -1,2 +1,9 @@
-# docs
-pumpkin.py documentation
+# [pumpkin.py](https://github.com/pumpkin-py/pumpkin-py) documentation
+
+## [Module installation](inst.md)
+
+## [Logs settings](log.md)
+
+## [ACL setup](acl.md)
+
+## [Functionalities setup & usage](func.md)
```

Comment:
I'd make this a listing, instead of H2 headers. And maybe write a sentence or two, so the users know what they will find on that page.